xcp_d/workflow/anatomical.py

```diff
@@ -255,8 +248,27 @@ def init_anatomical_wf(
     ------
     t1w : str
         Path to the T1w file.
-    t1seg : str
-        Path to the T1w segmentation file.
+        Only seems to be used for a stray branch of init_update_xform_wf.
+    t1w_to_template_xform
+    template_to_t1w_xform

```

Comment:
@madisoth our assumption is that the selected volumetric template from `collect_data` is applicable here. However, since the target space is fsLR, which uses MNI152NLin6Asym for its volumetric data, does the template in question _always_ have to be MNI152NLin6Asym?

Reply:
That's correct for all 4 of DCAN-HCP / HCP / fMRIPrep / NiBabies.


Supporting other, nonstandard volume spaces for CIFTI could help enable ingress from things like the DCAN macaque pipeline (HCP-based pipeline which outputs fsLR surfaces but non-MNI152 volumes) in the future I suppose.

Reply:
In that case, I think I need to anchor each CIFTI space we support to a specific volumetric space. At the moment, that would just be fsLR --> MNI152NLin6Asym, but I can try to write the code so it can be expanded easily.

Reply:
I've opened #720 for this.